packages/create-sitecore-jss/src/templates/nextjs-personalize/src/components/CdpPageView.tsx

```diff
@@ -2,6 +2,7 @@ import { useSitecoreContext, CdpHelper } from '@sitecore-jss/sitecore-jss-nextjs
 import { useEffect } from 'react';
 import config from 'temp/config';
 import { init } from '@sitecore/engage';
+import { PosResolver } from '../lib/pos-resolver';
```

Comment:
Can use "lib" alias here
```suggestion
import { PosResolver } from 'lib/pos-resolver';
```